keycloakClient/keycloakClient_test.go

```diff
@@ -0,0 +1,33 @@
+package keycloakClient
+
+import (
+	"testing"
+	"time"
+
+	"github.com/tozny/utils-go"
+)
+
+var (
+	cyclopsServiceHost  = utils.MustGetenv("TOZNY_CYCLOPS_SERVICE_HOST")
+	HTTPTimeout         = 10 * time.Second
+	keycloakMasterRealm = utils.MustGetenv("KEYCLOAK_MASTER_REALM")
+	keycloakUsername    = utils.MustGetenv("KEYCLOAK_USERNAME")
+	keycloakPassword    = utils.MustGetenv("KEYCLOAK_PASSWORD")
+)
+
+// Verifies that calling getToken with the master realm succeeds
+func TestGetTokenSucceedsWithValidClientCredentials(t *testing.T) {
+	keycloakClientConfig := Config{
+		AddrTokenProvider: cyclopsServiceHost,
+		AddrAPI:           cyclopsServiceHost,
+		Timeout:           HTTPTimeout,
+	}
+	kcClient, err := New(keycloakClientConfig)
+	if err != nil {
+		t.Fatalf("Failure creating keycloak client with err: %+v", err)
+	}
+	_, err = kcClient.GetToken(keycloakMasterRealm, keycloakUsername, keycloakPassword)
```

Comment:
nit: wouldn't hurt to also test/assert that the returned token is a non empty string